streams/json_from_readable_stream.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+
+import { textFromReadableStream } from "./text_from_readable_stream.ts";
+
+export function jsonFromReadableStream(
+  readableStream: ReadableStream,
+): Promise<unknown> {
+  return textFromReadableStream(readableStream).then((res) => {
+    console.log(res);
```

Comment:
```suggestion
```

Reply:
Oh haha, forgot I left that in there. Nice catch.